src/js/node/http.ts

```diff
@@ -155,6 +155,10 @@ var FakeSocket = class Socket extends Duplex {
 
   _final(callback) {}
 
+  get encrypted() {
```

Comment:
instead of a getter, you should just set the `encrypted` property directly on it

Reply:
👍